### DISCUSSION TITLE
🚨 Breaking change on Resy 🚨

### DISCUSSION THREAD

**Discussion by Alkaar** [Alkaar/resy-booking-bot#92] [Announcements]
If you've noticed that you've been unable to snipe reservations with the bot the past few days, the reason is below. 👇 

![resy-update](https://user-images.githubusercontent.com/6472561/222918431-c1fb369b-0ddd-40b3-bf42-0f8cee662a99.png)

This breaks the bot since the bot doesn't know your credit card info as it pulls the payment id from your profile. 💥 
This requires a one-time fix on your end to continue using the bot. See the instructions below. 🛠️ 

1. Find a restaurant that requires a credit card with an available reservation to book that you can cancel with no fees. 🔍 
2. Book the reservation 🔫 
3. When you try to confirm the reservation, it will prompt you to re-enter your credit card info. 💸 
4. Finish booking the reservation 🏁 
5. Cancel the reservation 🚫 

That's it! You can't remove and re-add your credit card. It throws an error. 🤨 
Comment below if anyone is having any issues with this. ✍️

**Comment by tkershaw1** [comment]
I tried this and it keeps saying I missed the shot when I test with tables I know are available. Any suggestions? I can see in the logs that it is finding the table, but just saying they aren't available even though I know they are

**Reply by Alkaar** [reply]
Your issue I believe is unrelated to this post. You can go to https://github.com/Alkaar/resy-booking-bot/issues/102 where I will post updates.
